test/conformance/service_test.go

```diff
@@ -396,6 +398,20 @@ func TestReleaseService(t *testing.T) {
 		[]string{expectedFirstRev, expectedSecondRev},
 		[]string{"latest", "candidate", "current"},
 		[]string{expectedThirdRev, expectedSecondRev, expectedFirstRev})
+
+	// Now update the service to use `@latest` as candidate.
+	revisions[1] = v1alpha1.ReleaseLatestRevisionKeyword
+	logger.Info("Updating Service to split traffic between two `current` and `@latest`")
+	if objects.Service, err = test.PatchReleaseService(logger, clients, objects.Service, revisions, 50); err != nil {
+		t.Fatalf("Service %s was not updated to release: %v", names.Service, err)
+	}
+
+	// The domains should not change, since configuration was not changed.
+	validateDomains(t, logger, clients,
+		names.Domain,
+		[]string{expectedFirstRev, expectedSecondRev},
```

Comment:
I don't think this is right...

Your traffic should be split between the first and third revision. This is checking that the traffic is split between the first and second revision.

I think this is passing because the validation happens faster than our traffic adjustment.